documentation/source/user_section/command-line.rst

```diff
@@ -13,117 +13,174 @@ Command-Line Tools
+- sct_dmri_display_bvecs_ - Display scatter plot of gradient directions from bvecs file.
 - sct_dmri_moco_ - Slice-wise motion correction of DWI data.
 - sct_dmri_separate_b0_and_dwi_ - Separate b=0 and DW images from diffusion dataset.
 - sct_dmri_transpose_bvecs_ - Transpose bvecs file.
 
-Magnetization transfer:
+Magnetization transfer
+======================
 
-- sct_compute_mtr_ - Register image without (MTC0) and with magnetization transfer contrast (MTC1) and compute MTR
+- sct_compute_mtr_ - Compute magnetization transfer ratio (MTR).
+- sct_compute_mtsat_ - Compute MTsat and T1map `[Helms et al. Magn Reson Med 2008]
+  <https://pubmed.ncbi.nlm.nih.gov/19025906/>`_.
 
-Functional MRI:
+Functional MRI
+==============
 
 - sct_fmri_compute_tsnr_ - Compute the temporal signal-to-noise ratio from fMRI nifti files.
 - sct_fmri_moco_ - Correct fMRI data for motion.
 
-Miscellaneous:
+Metric processing
+=================
 
-- sct_check_dependencies_ - Check installation and compatibility of SCT.
-- sct_testing_ - Runs complete testing to make sure SCT is working properly.
-- sct_compute_ernst_angle_ - Compute Ernst angle.
-- sct_dice_coefficient_ - Compute 2D or 3D DICE coefficient between two binary images.
+- sct_analyze_texture_ - Extraction of grey level co-occurence matrix (GLCM) texture features from an image within a
+  given mask.
+- sct_extract_metric_ - Estimate metric value within tracts, taking into account partial volume effect.
+
+Image manipulation
+==================
+
+- sct_convert_ - Convert image file to another type.
+- sct_crop_image_ - Tools to crop an image, either via command line or via a Graphical User Interface (GUI).
+- sct_denoising_onlm_ - Utility function to denoise images.
 - sct_flatten_sagittal_ - Flatten the spinal cord in the sagittal plane (to make nice pictures).
-- sct_label_utils_ - Utility function for label images.
 - sct_image_ - Performs various operations on images (split, pad, etc.).
 - sct_maths_ - Performs mathematical operations on images (threshold, smooth, etc.).
+- sct_merge_images_ - Merge images to the same space.
+- sct_resample_ - Anisotropic resampling of 3D or 4D data.
+- sct_smooth_spinalcord_ - Smooth the spinal cord along its centerline.
+
+Miscellaneous
+=============
 
+- sct_compute_ernst_angle_ - Compute Ernst angle.
+- sct_compute_snr_ - Compute SNR using methods described in `[Dietrich et al. JMRI 2007]
+  <https://pubmed.ncbi.nlm.nih.gov/17622966/>`_.
+- sct_download_data_ - Download binaries from the web.
+- sct_qc_ - Generate Quality Control (QC) report following SCT processing.
+- sct_run_batch_ - Wrapper to processing scripts, which loops across subjects.
+
+System tools
+============
+
+- sct_check_dependencies_ - Check installation and compatibility of SCT.
+- sct_testing_ - Runs complete testing to make sure SCT is working properly.
+- sct_version_ - Display SCT version.
 
 
 Main Tools
```

Comment:
I'm noticing that the inline commands show an error message when rendered on this page, e.g. https://spinalcordtoolbox.com/en/jca-1656-rtd/user_section/command-line.html#sct-analyze-lesion. 

```
System Message: ERROR/6 (/home/docs/checkouts/readthedocs.org/user_builds/spinalcordtoolbox/checkouts/jca-1656-rtd/documentation/source/user_section/command-line.rst, line 131)

Command 'sct_analyze_lesion -h' failed: [Errno 2] No such file or directory: 'sct_analyze_lesion': 'sct_analyze_lesion'
```

Is this expected behavior?

Reply:
No, we are trying to use a sphinx plugin to run a command (in this case a `sct_command -h`) to show the help message of the specific command, but this won't run on the RTD build system since sct hasn't been installed.

The install of sct is not a simple `pip install`, therefore we need a strategy on how to deal with this.

Reply:
hum... this is strange. It also happens on the master branch (which was working before): 

![image](https://user-images.githubusercontent.com/2482071/89562302-c2923380-d7e7-11ea-8eeb-2ec103bf63ef.png)

so i'm wondering if this is a temporary glitch in RTD...?

Reply:
> No, we are trying to use a sphinx plugin to run a command (in this case a `sct_command -h`) to show the help message of the specific command, but this won't run on the RTD build system since sct hasn't been installed.
> 
> The install of sct is not a simple `pip install`, therefore we need a strategy on how to deal with this.

ah! so i was wrong @Drulex : this has never worked so far on the master branch (from RTD)?

Reply:
now that we have the scripts listed in the setup entry points, this _should_ work with a pip install by RTD. I'll do some testing...

Reply:
From what I recall no. It was first brought up here: https://github.com/neuropoly/spinalcordtoolbox/pull/2781#issuecomment-658818129 then I guess it fell in the cracks.

Reply:
ok, i figured the proper config for .readthedocs.yml, which installs the scripts, however now i'm bumping in the following issue:
~~~
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/spinalcordtoolbox/envs/jca-1656-rtd/bin/sct_analyze_lesion", line 5, in <module>
    from spinalcordtoolbox.compat.launcher import main
  File "/home/docs/checkouts/readthedocs.org/user_builds/spinalcordtoolbox/envs/jca-1656-rtd/lib/python3.6/site-packages/spinalcordtoolbox/__init__.py", line 3, in <module>
    from .utils import __version__, __sct_dir__, __data_dir__, __deepseg_dir__
  File "/home/docs/checkouts/readthedocs.org/user_builds/spinalcordtoolbox/envs/jca-1656-rtd/lib/python3.6/site-packages/spinalcordtoolbox/utils.py", line 462, in <module>
    __version__ = _version_string()
  File "/home/docs/checkouts/readthedocs.org/user_builds/spinalcordtoolbox/envs/jca-1656-rtd/lib/python3.6/site-packages/spinalcordtoolbox/utils.py", line 444, in _version_string
    install_type, sct_commit, sct_branch, version_sct = _git_info()
  File "/home/docs/checkouts/readthedocs.org/user_builds/spinalcordtoolbox/envs/jca-1656-rtd/lib/python3.6/site-packages/spinalcordtoolbox/utils.py", line 437, in _git_info
    with io.open(os.path.join(__sct_dir__, 'version.txt'), 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/docs/checkouts/readthedocs.org/user_builds/spinalcordtoolbox/envs/jca-1656-rtd/lib/python3.6/site-packages/version.txt'
~~~

which i will solve by adding a copy of the version.txt via the setup.py

Reply:
ok, it seems the problem is more fundamental: in `spinalcordtoolbox.utils`, the version is fetched using `os.path.join(__sct_dir__, 'version.txt')`, which, for a pip-installed package, is pointing to `envs/sct-env/lib/python3.6/site-packages/version.txt`. 

By adding this line:
https://github.com/neuropoly/spinalcordtoolbox/blob/734602113cdea1ad1d687ec82c24b1c4e8ff21c2/setup.py#L44

the file `version.txt` is copied under `envs/sct-env/version.txt`, therefore it does not solve the problem.

A possibly more elegant solution would be to edit `spinalcordtoolbox.utils` so it points to the `version.txt` file in site-package.

@Drulex @joshuacwnewton does that sounds like a reasonable approach? if so i'll open an issue

Reply:
ok i went ahead: https://github.com/neuropoly/spinalcordtoolbox/issues/2832